app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/SearchSoftwarePage.scala.html

```diff
@@ -42,17 +48,182 @@
   ))
 }
 
+@customSummaryListRow(key: String, filtersAvailable: Seq[VendorFilter]) = @{
```

Comment:
I think there might be some confusion here, because what I'm reading from your comment doesn't seem to align with what the code is actually doing.
Lets discuss further.